tests/tests_pytorch/checkpointing/test_legacy_checkpoints.py

```diff
@@ -45,7 +46,12 @@ def test_load_legacy_checkpoints(tmp_path, pl_version: str):
         assert path_ckpts, f'No checkpoints found in folder "{PATH_LEGACY}"'
         path_ckpt = path_ckpts[-1]
 
-        model = ClassificationModel.load_from_checkpoint(path_ckpt, num_features=24)
+        # legacy load utility added in 1.5.0 (see https://github.com/Lightning-AI/pytorch-lightning/pull/9166)
+        if pl_version == "local":
```

Comment:
This is the simplest way that I could think of ensuring we continue testing the legacy checkpoints. Another way could be to use [`torch.serialization.add_safe_globals`](https://docs.pytorch.org/docs/main/notes/serialization.html#torch.serialization.add_safe_globals), but it seems a little more complicated (particularly since we're using the `pl_legacy_patch` context manager already.